test/integration/features/coolstore.feature

```diff
@@ -0,0 +1,157 @@
+@coolstore
+Feature: Cool Store
+  In order to test Minishift under load
+  I need to setup a test environment
+  Using CentOS as image and 4G of memory
+  After which I will import image streams
+  Finally I will deploy the Cool Store
+  During this deployment I will measure memory and CPU usage
+
+
```

Comment:
At least for me, it is breaking the consistency and increasing the file length by one :) 
It looks nice with 1 line too :)